.github/workflows/playwright_pr_manual.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         shell: bash
     environment: PRE
     timeout-minutes: 280
-    runs-on: windows-latest
+    runs-on: windows-2022
```

Comment:
I totally agree with this change, but have we run the test (or part of it) to preview the possible failures? Do we prefer to face the reality tomorrow? :P